backend/app/tests/api/routes/test_google_oauth.py

```
@@ -45,236 +51,479 @@ def client(app_for_google_oauth_module: FastAPI, mock_db_session_fixture: MagicM
 @patch("app.api.routes.google_oauth.settings")
 def test_google_callback_success_new_user(
-    mock_route_settings: MagicMock, mock_requests_module: MagicMock, mock_create_access_token: MagicMock,
-    mock_crud_module: MagicMock, MockUserModel: MagicMock,
-    client: TestClient, mock_db_session_fixture: MagicMock
+    mock_route_settings: MagicMock,
+    mock_requests_module: MagicMock,
+    mock_create_access_token: MagicMock,
+    mock_crud_module: MagicMock,
+    MockUserModel: MagicMock,
+    client: TestClient,
+    mock_db_session_fixture: MagicMock,
```

Ruff (0.11.9)

Unused function argument: `mock_db_session_fixture`

(ARG001)